src/SFA.DAS.EAS.Portal.Client/Application/Queries/GetAccountQuery.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.CosmosDb;
+using SFA.DAS.EAS.Portal.Client.Data;
+using SFA.DAS.EAS.Portal.Client.Models.Concrete;
+
+namespace SFA.DAS.EAS.Portal.Client.Application.Queries
+{
+    internal class GetAccountQuery
+    {
+        private readonly IAccountsReadOnlyRepository _accountsReadOnlyRepository;
+
+        public GetAccountQuery(IAccountsReadOnlyRepository accountsReadOnlyRepository)
+        {
+            _accountsReadOnlyRepository = accountsReadOnlyRepository;
+        }
+
+        public async Task<AccountDto> Get(long accountId, CancellationToken cancellationToken = default)
```

Comment:
Be better to stream the Account ReadModel direct from the cosmos database.  Add as tech debt in 454.

Reply:
I'm not sure what you mean by streaming it direct. Could you please explain, or provide a link for me to read?